src/app/services/bucket/qarnotBucketService.py

```diff
@@ -0,0 +1,26 @@
+from qarnot import connection, bucket
+import os
+
+from app.services.bucket.baseBucketService import BaseBucketService
+from app import config as cfg
+
+
+class QarnotBucketService(BaseBucketService):
+
+    async def upload_file(self, bucket_name: str, bucket_key: str, file_binary: bin):
+        conn = connection.Connection(client_token=cfg.QARNOT_TOKEN)
+        media_bucket = bucket.Bucket(conn, bucket_name)
+        return await media_bucket.add_file(bucket_name, file_binary, bucket_key)
+
+    async def get_uploaded_file(self, bucket_name: str, bucket_key: str):
+        conn = connection.Connection(client_token=cfg.QARNOT_TOKEN)
+        media_bucket = bucket.Bucket(conn, bucket_name)
+        return await media_bucket.get_file(bucket_name, bucket_key)
+
+    async def fetch_bucket_files(self, bucket_name: str):
+        conn = connection.Connection(client_token=cfg.QARNOT_TOKEN)
```

Comment:
those two lines are used in every function. Maybe Create  a private function "connet_to_bucket" or "get_bucket" or something like that.
If we refactor the connection mechanism, it will be easier to propagate the changes.